src/main/java/core/basesyntax/io/impl/CsvReportSerializer.java

```
@@ -0,0 +1,39 @@
+package core.basesyntax.io.impl;
+
+import core.basesyntax.io.ReportSerializer;
+import core.basesyntax.model.FruitType;
+import core.basesyntax.model.Report;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvReportSerializer implements ReportSerializer {
+    @Override
+    public void save(Report report, Path path) {
+        List<String> csvRows = createCsvRows(report);
+        try (BufferedWriter writer = Files.newBufferedWriter(path)) {
+            for (String line : csvRows) {
+                writer.write(line);
+                writer.newLine();
+            }
+            writer.flush();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

This catch rethrows the IOException as `new RuntimeException(e)` without any informative message or path context. This violates the checklist guidance: "Don't ignore exceptions in try-catch blocks ... Let's rethrow a RuntimeException with an **informative** message and exception object." Provide an informative message and include the `path` variable and the original exception as the cause, e.g. `throw new RuntimeException("Can't save report to path: " + path, e);`. Quote from checklist: "Don't ignore exceptions in try-catch blocks ... Let's rethrow a RuntimeException with an **informative** message and exception object."